woocommerce-gateway-stripe.php

```diff
@@ -366,6 +366,10 @@ public function install() {
 					add_woocommerce_inbox_variant();
 					$this->update_plugin_version();
 
+					// Add webhook reconfiguration
+					$account = self::get_instance()->account;
+					$account->reconfigure_webhooks_on_update();
```

Comment:
I suppose we could be more restrictive here. Only reconfigure if there's an existing webhook and it listens to a different set of `enabled_events`.

Reply:
I agree... it's a bit more work, but it makes sense to only update the existing webhook if needed; we won't be adding events frequently.

Reply:
@diegocurbelo Updated in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3853/commits/7a42ac72ab994be00867d4e0872bfe43f4afc93b :)